webapp/store/auth.js

```diff
@@ -72,32 +73,7 @@ export const actions = {
     const client = this.app.apolloProvider.defaultClient
     const {
       data: { currentUser },
-    } = await client.query({
-      query: gql`
-        query {
-          currentUser {
-            id
-            name
-            slug
-            email
-            avatar
-            role
-            about
-            locationName
-            locale
-            contributionsCount
-            commentedCount
-            allowEmbedIframes
-            showShoutsPublicly
-            termsAndConditionsAgreedVersion
-            socialMedia {
-              id
-              url
-            }
-          }
-        }
-      `,
-    })
+    } = await client.query({ query: currentUserQuery })
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-03-05T08:50:51Z" title="Thursday, March 5th 2020, 9:50:51 am +01:00">Mar 5, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -72,32 +73,7 @@ export const actions = {
     const client = this.app.apolloProvider.defaultClient
     const {
       data: { currentUser },
-    } = await client.query({
-      query: gql`
-        query {
-          currentUser {
-            id
-            name
-            slug
-            email
-            avatar
-            role
-            about
-            locationName
-            locale
-            contributionsCount
-            commentedCount
-            allowEmbedIframes
-            showShoutsPublicly
-            termsAndConditionsAgreedVersion
-            socialMedia {
-              id
-              url
-            }
-          }
-        }
-      `,
-    })
+    } = await client.query({ query: currentUserQuery })
```


Nice that you moved the query to `graphql/User.js`. 😍